proto/p4/config/p4info.proto

```diff
@@ -78,17 +78,22 @@ message Table {
   repeated int32 action_ids = 3;
   // 0 (default value) means that the table does not have a const default action
   int32 const_default_action_id = 4;
+  // a table may have a const default action, whose action parameter values can
+  // be changed at runtime. However, in most cases a table's action entry is set
+  // in the P4 program and cannot be changed, which is what this boolean flag
+  // indicates.
+  bool has_const_default_action_entry = 5;
```

Comment:
I like your proposed name so I'll update the pull request.

Regarding the second part of your comment, I believe this is a **P4** problem that has not been solved: the ability to rename actions in the scope of a table when binding parameters at compile-time.